src/app/auth/login/login.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component} from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7138913)
```suggestion
import { FormBuilder, FormGroup, Validators } from "@angular/forms";
```